Automattic/wp-calypso
# 500: drop pug in favor of jsx

Part of _Eliminate Pug_ (#19822). Removes the Pug.js template which is used for the 500 response.

### Testing

1. You need to throw an error on the server side e.g. in `/server/render/index.js`:

```diff
diff --git a/server/render/index.js b/server/render/index.js
index 9fcf7059c4..94304a278d 100644
--- a/server/render/index.js
+++ b/server/render/index.js
@@ -159,6 +159,8 @@ export function serverRender( req, res ) {
        context.head = { title, metas, links };
        context.config = config.ssrConfig;

+       throw new Error( 'gotcha' );
+
        if ( config.isEnabled( 'desktop' ) ) {
                res.render( 'desktop', context );
                return;
```

2. Go to e.g. https://calypso.localhost:3000 and check the page source (Right-click > "View page source")
3. Make sure the output from this branch is not significantly different to the `master` branch.
